.github/workflows/production.yml

```diff
@@ -1,7 +1,13 @@
 ---  
 name: Release to Production
 on: 
+   repository_dispatch:
    workflow_dispatch:
+    inputs:
+      tags:
+        description: 'Release Tag'
+        required: true
+        default: 'master'
```

Comment:
Can this be blank, or the default not set. 

I'd like to see releasing from `master` _very_ strongly discouraged - we should always be releasing a tag from testing

Reply:
I am going to remove it as you suggest and I am going to also add a check so it errors if you do chose master. I will look at a way to see if the tag is in the list of releases, so you cannot choose any old branch